src/structures/ThreadChannel.js

```diff
@@ -135,12 +135,13 @@ class ThreadChannel extends Channel {
   }
 
   /**
-   * The time the thread was last archived or unarchived at
-   * @type {?Date}
+   * The time when the thread's archive status was last changed
+   * <info>If the thread was never archived or unarchived, this is the same as {@link ThreadChannel#createdAt}</info>
```

Comment:
Should I divide this like `archiveTimestamp` even if it's in the characters limit?